cmd/selfupdate/selfupdate.go

```diff
@@ -169,7 +169,7 @@ func InstallUpdate(ctx context.Context, opt *Options) error {
 		} else {
 			err := installPackage(ctx, opt.Beta, newVersion, siteURL, opt.Package)
 			if err == nil {
-				fmt.Printf("Successfully updated rclone package from version %s to version %s\n", oldVersion, newVersion)
+				fs.Infof(nil, "Successfully updated rclone package from version %s to version %s\n", oldVersion, newVersion)
```

Comment:
fs.Infof does not need trailing "\n", please remove it

Reply:
ok, fixed